rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/HelmBakeTemplateUtils.java

```diff
@@ -126,12 +126,17 @@ protected List<String> buildOverrideList(Map<String, Object> overrides) {
         .collect(Collectors.toList());
   }
 
+  /** Accessor for whether to expand artifact reference URIs */
+  protected boolean isExpandArtifactReferenceURIs() {
+    return (artifactStore != null && helmConfig.isExpandOverrides());
```

Comment:
Can you explain the use case for this a little bit? Im unsure why you want to not expand something in rosco? Or maybe Im misunderstanding how this works.

Reply:
Before this change, artifact references would sometimes remain in the values file, e.g.:
```
Created overrides file at /tmp/rosco-9390928878072864271/overrides_d07566cf-e592-4b43-ad94-d947782f4a88.yml with the following contents:
---
my-override: "ref://dbyrontest/7678d0be07b31c883c436a70522c58ec0f724f1bf8bbeb293165e330f495e321"
```

Reply:
The override in question was configured as:
```
"my-override": "${#stage(\"Bake (Manifest)\").outputs.resolvedExpectedArtifacts[0].boundArtifact.reference}"
```